src/style.css

```diff
@@ -1 +1,105 @@
 /* add styles here */
+* {
+  box-sizing: border-box;
+  border: 0;
+
+  font-weight: 300;
+}
+
+.bar {
+  position: relative;
+  background: #ffff;
+
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  height: 70px;
+  width: auto;
+  margin: 20px 0 0;
+  padding: 0;
+}
+
+.bar--small {
+  height: 42px;
+}
+
+.bar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.button {
+  width: 19px;
+  height: 19px;
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  background-color: transparent;
+  margin: 0;
+  padding: 0;
+  border: 1px solid transparent;
+}
+
+.button--small {
+  width: 11px;
+  height: 11px;
+  left: 13px;
+  top: 15px;
+}
+
+.button--image {
+  position: absolute;
+  left: 0;
+  top: 0;
+  margin: 0;
+  padding: 0;
+  border: 0;
+}
+
+.button--image_small {
+  width: 11px;
+  height: 11px;
+}
+
+.input {
+  height: 19px;
+  width: calc(100% - 62px);
+  position: absolute;
+  top: 25px;
+  left: 62px;
+  margin: 0;
+  padding: 0;
+  border: 0;
+  background-color: transparent;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.input::placeholder {
+  color: #3d4e61;
+}
+
+.input--small {
+  width: calc(100% - 33px);
+  left: 33px;
+  top: 12px;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 20px;
+}
+
+.bar:focus-within {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.input:focus {
+  font-weight: 800;
+  outline: none;
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+}
+
+@font-face {
```

Comment:
Remember to use fallback fonts. It's important to have an alternative font-family in case the main one doesn't work.